Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.yml

```diff
@@ -87,6 +87,12 @@ configuration:
   name: debug_mode
   type: 8
   required: false
+- display: Drop Playbook ID
+  name: drop_playbookId
+  defaultvalue: "false"
+  type: 8
+  required: false
+  additionalinfo: 'Should the integration mirror the playbook ID field to the remote system? Note: Mirroring the playbook ID means that the remote system must be running the same playbook as the source system.'
```

Comment:
please add documentation in the readme that explains the case that this is used.